src/test/kotlin/com/terraformation/backend/customer/db/ProjectStoreTest.kt

```diff
@@ -210,7 +210,7 @@ class ProjectStoreTest : DatabaseTest(), RunsAsUser {
       store.update(projectId) {
         ExistingProjectModel(
             description = "New description",
-            id = ProjectId(-1),
+            id = projectId,
```

Comment:
Can probably remove this change -- the intent of the original test was to make sure any changes to the project ID in the returned model would be ignored.